wordpress-mobile/WordPress-Aztec-iOS
# Add support for HTML comment nodes.

How to test:

 - Start the demo app
 - Switch between Visual and HTML mode
 - see if the  tag is being converted properly
 - add a new comment to the HTML view, switch to visual mode, and them back to visual

Some questions to answer in the future:

 - Should comments be visible in Visual mode?
 - Should we allow editing or deleting in visual mode?
 - We still need to add an extra case for the  to be parsed visually as an separator.
